.semaphore/release.yml

```
@@ -12,14 +12,18 @@ blocks:
           value: "on"
       secrets:
         - name: github-release-bot-agent
+        - name: semaphorecipushrobot
```

```suggestion
        - name: dockerhub-write
```
this is the new name for this secret